.github/actions/functest/action.yml

```diff
@@ -91,10 +91,10 @@ runs:
         shell: ${{ env.SHELL }}
         run: |
           tests all  --cross-prefix="${{ env._CROSS_PREFIX }}" --cflags="${{ inputs.cflags }}" --opt=${{ inputs.opt }} --${{ env.FUNC }} --${{ env.KAT }} --${{ env.NISTKAT }} --${{ env.ACVP }} -v
-      - name: Check namespacing ${{ inputs.mode }} ${{ inputs.opt }} tests (${{ env.FUNC }}, ${{ env.KAT }}, ${{ env.NISTKAT }})
-        shell: ${{ env.SHELL }}
-        run: |
-          check-namespace
+      # - name: Check namespacing ${{ inputs.mode }} ${{ inputs.opt }} tests (${{ env.FUNC }}, ${{ env.KAT }}, ${{ env.NISTKAT }})
```

Comment:
the namespacing test runs on the object files. With the changes proposed here, there are no object files anymore, so there is nothing to check. 
The `assert checked > 0` is there for exactly that reason: To fail in case it is actually a no-op. 
We'll have to reimplement the namespacing check.